tasmanian/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+find_package(Tasmanian 7.5 PATHS "${TASMANIAN_DIR}" REQUIRED)
```

Comment:
Move this to the top level CMakeLists file like the other packages (and guard with `ENABLE_TASMANIAN`).

Reply:
Limited development access at the moment, will update when I get a chance. Thanks for commenting.

Reply:
Done!